src/bundle/Controller/ContentViewController.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * File containing the ContentViewController class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace EzSystems\HybridPlatformUiBundle\Controller;
+
+use eZ\Publish\Core\MVC\Symfony\Controller\Controller;
+use eZ\Publish\Core\MVC\Symfony\View\ContentView;
+
+class ContentViewController extends Controller
+{
+    public function versionsTabAction(ContentView $view)
+    {
+        $contentInfo = $view->getContent()->getVersionInfo()->getContentInfo();
+        $contentService = $this->getRepository()->getContentService();
+        $versions = $contentService->loadVersions($contentInfo);
+
+        $userService = $this->getRepository()->getUserService();
+        $authors = [];
+        foreach ($versions as $version) {
+            $authors[$version->id] = $userService->loadUser($version->creatorId);
```

Comment:
here as well, same remark as inhttps://github.com/ezsystems/hybrid-platform-ui/pull/18#discussion_r122208466 (so ping @SylvainGuittard to define the expected behavior)